Yoast/wordpress-seo
# Simple compat for Genesis title and description element filters

## Summary

The problem is that Genesis automatically disables it's SEO features and with that also disables site-title handling. With this minor patch the user can select the title and description elements for the site header within Yoast's plugin.
## Relevant technical choices:

*
## Test instructions

This PR can be tested by following these steps:
- Install a Genesis based theme and change the element options. These changes will be visible in the frontend HTML.

Fixes #

@see Slack - WPNL -> #genesis
